pkg/code-generator/collector/executor.go

```diff
@@ -105,10 +109,11 @@ func (o *OpenApiProtocExecutor) Execute(protoFile string, toFile string, imports
 	_ = os.Mkdir(directoryPath, os.ModePerm)
 
 	cmd.Args = append(cmd.Args,
-		fmt.Sprintf("--openapi_out=yaml=true,single_file=false,include_description=%v,enum_as_int_or_string=%v,additional_empty_schema=%v:%s",
+		fmt.Sprintf("--openapi_out=yaml=true,single_file=false,include_description=%v,enum_as_int_or_string=%v,additional_empty_schema=%v,disable_kube_markers=%v:%s",
```

Comment:
I like this change, because it follows the precedent that existed. An alternative way to perhaps improve this "API" would be to just expose a raw string to users. This string would be passed from callers of the code, down to protoc. This way, the next time we add an argument to our open-api protoc impelementation, we don't need to pipe through values, and can instead just update the string

Reply:
I'm not sure I prefer that to exposing a clear option in solo-kit that IDEs won't let you typo and such, but certainly a conversation we can have.